internal/executor/build/task.go

```diff
@@ -89,11 +89,11 @@ func (task *Task) ProtoCommands() []*api.Command {
 	return result
 }
 
-func (task *Task) UniqueName() string {
+func (task *Task) UniqueDescription() string {
 	if len(task.Labels) == 0 {
-		return task.Name
+		return fmt.Sprintf("'%s' Task", task.Name)
```

Comment:
Perhaps the `Task` here and below should really be `task` to be consistent with the script type names?

Reply:
`'Tests (Go 1.15) Task` looks better then `'Tests (Go 1.15) task`. Maybe I can check if task name is capitalized then have a consistent task or Task. What do you think?